src/frontend/cunningham.cjs

```diff
@@ -65,8 +65,18 @@ module.exports = {
         },
       },
       components: {
+        tabs: {
+          'border-bottom-color': '#E7E8EA', // --c--theme--colors--greyscale-300
+        },
         button: {
           'font-family': 'Montserrat',
+          tab: {
+            'default-color': '#555F6B', // --c--theme--colors--greyscale-700
+            'hover-color': '#0C1A2B', // --c--theme--colors--greyscale-900
+            'selected-color': '#0556BF', // --c--theme--colors--primary-600
+            'focus-color': '#555F6B', // --c--theme--colors--greyscale-300
+            'disabled-color': '#79818A', // --c--theme--colors--greyscale-600
+          },
```

Comment:
I do not see tab as a variant of button component. I'll put this property at the root level of `components`.

Reply:
In styles, you are not using button tokens so I'm not quit sure you'll have to copy button tokens.

Except about the border-radius to set it to 0 ? 🤔